src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/vcf/VcfFuncotationFactory.java

```diff
@@ -147,12 +147,20 @@ private FuncotationMetadata createFuncotationMetadata(final Path sourceFilePath)
     List<VCFInfoHeaderLine> createFuncotationVcfInfoHeaderLines(final VCFHeader vcfHeader) {
         final List<VCFInfoHeaderLine> supportedVcfInfoHeaderLines = vcfHeader.getInfoHeaderLines().stream()
                 .filter(vcfInfoHeaderLine -> supportedFieldNames.contains(createFinalFieldName(name, vcfInfoHeaderLine.getID())))
+                .map(vcfInfoHeaderLine -> copyWithRename(vcfInfoHeaderLine, name))
                 .collect(Collectors.toList());
 
+        // Add in the ID field to the meta data:
+        final VCFInfoHeaderLine idHeaderLine = new VCFInfoHeaderLine(
+                createFinalFieldName(name, "ID"),
```

Comment:
Can you make this a constant?  Or grab an existing constant?

Reply:
I'll make it a constant.  I want to be explicit with field names.